packages/gitbook/src/components/AIChat/AIChatButton.tsx

```diff
@@ -30,7 +32,15 @@ export function AIChatButton(props: {
                     <KeyboardShortcut keys={['mod', 'i']} className="border-tint-11 text-tint-1" />
                 </div>
             }
-            onClick={() => assistant.open()}
+            onClick={() => {
+                setSearchState((prev) => ({
```

Comment:
This looks messy; I think we are in a mess with state management and create more mess to get out of the mess.

If you explain what we want with the URL / state; I might find a solution